paper/paper.bib

```diff
@@ -0,0 +1,151 @@
+  title =        "The Role of Myocardial Fiber Direction in Epicardial Activation Patterns via Uncertainty Quantification ",
+  booktitle =    "Computing in Cardiology",
+  month =        sep,
+  year =         "2021",
+  doi =          "",
+  volume =       "48"
+}
+
+@InProceedings{JDT:Ber2021,
+  author =       "Jake Bergquist and Brian Zenger and  Lindsay Rupp and Akil Narayan and Jess Tate and Rob MacLeod",
+  title =        "Uncertainty Quantification in Simulations of Myocardial Ischemia",
+  booktitle =    "Computing in Cardiology",
+  month =        sep,
+  year =         "2021",
+  doi =          "",
+  volume =       "48"
+}
+
+@article{JDT:Tat2021c,
+	author = {Tate, Jess and Rampersad, Sumientra and Charlebois, Chantel and Liu, Zexin and Bergquist, Jake and White, Dan and Rupp, Lindsay and Brooks, Dana and Narayan, Akil and MacLeod, Rob},
+	booktitle = {Brain Stimulation: Basic, Translational, and Clinical Research in Neuromodulation},
+	date = {2021/11/01},
+	doi = {10.1016/j.brs.2021.10.226},
+	isbn = {1935-861X},
+	journal = {Brain Stimulation: Basic, Translational, and Clinical Research in Neuromodulation},
+	month = {2022/01/04},
+	number = {6},
+	pages = {1659--1660},
+	publisher = {Elsevier},
+	title = {Uncertainty Quantification in Brain Stimulation using UncertainSCI},
+	ty = {JOUR},
+	url = {https://doi.org/10.1016/j.brs.2021.10.226},
+	volume = {14},
+	year = {2021}
+	}
+
+@book{ACN:Xiu2010,
+	title = {Numerical {Methods} for {Stochastic} {Computations}: {A} {Spectral} {Method} {Approach}},
+	isbn = {0-691-14212-2},
+	shorttitle = {Numerical {Methods} for {Stochastic} {Computations}},
+	publisher = {Princeton University Press},
+	author = {Xiu, Dongbin},
+	month = jul,
+	year = {2010}
+}
+
+@article{ACN:Guo2018,
+	title = {Weighted {Approximate} {Fekete} {Points}: {Sampling} for {Least}-{Squares} {Polynomial} {Approximation}},
+	volume = {40},
+	issn = {1064-8275},
+	url = {http://epubs.siam.org/doi/abs/10.1137/17M1140960},
+	doi = {10.1137/17M1140960},
+	number = {1},
+	journal = {SIAM Journal on Scientific Computing},
+	author = {Guo, L. and Narayan, A. and Yan, L. and Zhou, T.},
+	year = {2018},
+	note = {arXiv:1708.01296 [math.NA]},
+	pages = {A366--A387}
+}
+
+@article{ACN:Coh2017,
+	title = {Optimal weighted least-squares methods},
+	volume = {3},
+	issn = {2426-8399},
+	url = {http://smai-jcm.cedram.org/item?id=SMAI-JCM_2017__3__181_0},
+	doi = {10.5802/smai-jcm.24},
+	journal = {SMAI Journal of Computational Mathematics},
+	author = {Cohen, Albert and Migliorati, Giovanni},
+	year = {2017},
+	note = {arxiv:1608.00512 [math.NA]},
+	pages = {181--203}
+}
+
+@article{ACN:Nar2018,
+	title = {Computation of induced orthogonal polynomial distributions},
+	volume = {50},
+	url = {https://epub.oeaw.ac.at?arp=0x003a184e},
+	doi = {10.1553/etna_vol50s71},
+	journal = {Electronic Transactions on Numerical Analysis},
+	author = {Narayan, Akil},
+	year = {2018},
+	note = {arXiv:1704.08465 [math]},
+	pages = {71--97}
+}
```

Comment:
@InProceedings{Ram2021,
	title = {Quantification of uncertainty due to tissue conductivity variability in simulations of brain stimulation},
	booktitle = {10th International IEEE EMBS Conference on Neural Engineering},
	author = {Rampersad S, Charlebois C, Tate J, MacLeod R, Brooks DH, Narayan A},
	year = {2021}}